datastore/README.rst

```diff
@@ -0,0 +1,184 @@
+RESTful Python Client for Google Cloud Datastore
+================================================
+
+|pypi| |pythons|
+
+Installation
+------------
+
+.. code-block:: console
+
+    $ pip install --upgrade gcloud-rest-datastore
+
+Usage
+-----
+
+We're still working on documentation; for now, this should help get you
+started:
+
+.. code-block:: python
+
+    from gcloud.rest.datastore import Datastore
```

Comment:
I'd like the examples to come directly from integration tests. I'll do that when I can write them.